### ISSUE TITLE
Program does not click anymore suddenly. 

### ISSUE THREAD

**Issue by Tazzetiel** [Aareon/NexusDownloadFlow-2023#2]
I made sure to replace the assets as instructed, but it seems to me like somehow the program still copies the original assets into the temp folder in app data, instead of my custom ones (it definitely does not copy my correct ones to the "AppData\Local\Temp\_MEI[random numbers each launch]\assets" folder, but the original ones. Replacing those in the temp area does not help either. 

The thing that confuses me is, it used to work perfectly fopr like damn near a day, before starting to fail suddenly.

**Comment by Aareon** [comment]
I honestly have no idea why this happened. I just tested the previous release that you were on and it worked even today, on a totally different computer.

However, this should be resolved regardless in the latest release. Going to close this now, please open a new issue if you discover anything else.
